### PR TITLE
add a deployment guide for Livebook

### PR DESCRIPTION
**Pull request by catflydotio**
### Summary of changes
Adds a deployment guide for Livebook which we can use to replace the web launcher

### CODE REVIEW

File: app-guides/livebook.html.markerb [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+<code>WARNING The app is not listening on the expected address and will not be reachable by fly-proxy.</code><br>
+<code>You can fix this by configuring your app to listen on the following addresses:</code><br>
+<code>  - 0.0.0.0:8080</code>
+
+If you've followed these instructions thoroughly, you can ignore this message. It's a false alarm caused by provisioning of the initial resources causing the startup time to take longer than flyctl expects. 
+</div>
+
+## Visit your Livebook app
+
+To use your livebook, visit `https://<your-app-name>.fly.dev` in the browser, or type `fly apps open`, and enter your password where prompted. Create a new notebook, and save it in the `/data` directory.
```
Comment (andie787) [inline app-guides/livebook.html.markerb:108]: Curious how you do this specifically when `/data` seems to already be the default for disk?

**Review by andie787 — COMMENTED** [review]
First of all: yay!
Left a few suggestions to consider.

File: app-guides/livebook.html.markerb [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+The `--ha=false` option tells `fly deploy` not to [create a second Machine for redundancy](/docs/reference/app-availability/#redundancy-by-default-on-first-deploy). Fly volumes don't automatically synchronize throughout an app, so the simplest way to run this app is as a single Machine, with the availability repercussions that entails.
+
+<div class="note icon">
+At the time of writing, you may see this error message as deployment finishes: 
+<code>WARNING The app is not listening on the expected address and will not be reachable by fly-proxy.</code><br>
+<code>You can fix this by configuring your app to listen on the following addresses:</code><br>
+<code>  - 0.0.0.0:8080</code>
+
+If you've followed these instructions thoroughly, you can ignore this message. It's a false alarm caused by provisioning of the initial resources causing the startup time to take longer than flyctl expects. 
+</div>
```
[the remark this answers is not in the document]
Comment (asib) [reply app-guides/livebook.html.markerb:104]: Nor did I!

File: app-guides/livebook.html.markerb [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+## Visit your Livebook app
+
+To use your livebook, visit `https://<your-app-name>.fly.dev` in the browser, or type `fly apps open`, and enter your password where prompted. Create a new notebook, and save it in the `/data` directory. 
+
+You can test that your notebook was saved onto the persistent volume by running `fly apps restart` and clicking "Open" on the Livebook home page to find and open it from storage.
+
+## A note on `auto_stop_machines` and `auto_start_machines` settings
+
+Note the `auto_stop_machines` and `auto_start_machines` config options set to `true` in the app config. The Fly proxy will shut down the Machine when there's nobody connected to it. This means if you forget to save your notebook and it's not autosaving, you could lose data if it shuts down.
```
[the remark this answers is not in the document]
Comment (catflydotio) [reply app-guides/livebook.html.markerb:114]: I think you have to give the notebook a name and save it explicitly the first time.

File: app-guides/livebook.html.markerb (hunk 2 above)
[the remark this answers is not in the document]
Comment (catflydotio) [reply app-guides/livebook.html.markerb:104]: It happened numerous times. Got it again this morning. Only on the first deployment, when it's provisioning a volume.

File: app-guides/livebook.html.markerb (hunk 1 above)
Comment (catflydotio) [reply app-guides/livebook.html.markerb:108]: There's a file browser in the Livebook.

image

You can go up a dir and save it in /app for example, where it wouldn't be persisted.

File: app-guides/livebook.html.markerb (hunk 2 above)
[the remark this answers is not in the document]
Comment (catflydotio) [reply app-guides/livebook.html.markerb:104]: yyz

**Pull request merged**